docs/validators/sigstore_cosign.md

```diff
@@ -7,13 +7,9 @@ You can read more about it [here](https://blog.sigstore.dev/).
 Connaisseur currently supports the elementary function of verifying Cosign-generated signatures against the locally created corresponding public keys.
 We plan to expose further features of Cosign and Sigstore in upcoming releases, so stay tuned!
 
-> :warning: Sigstore and Cosign are currently in *pre-release* state and under heavy development and so is our support for them. We therefore consider this an *experimental feature* that might unstable over time. As such, it is not part of our semantic versioning guarantees and we take the liberty to adjust or remove it with any version at any time without incrementing MAJOR or MINOR.
```

Comment:
Is our support for cosign not experimental anymore?

Reply:
imho I feel good about it and consider it tested and secure. Anything that strikes you as experimental?